client/src/utils/report-error.ts

```diff
@@ -1,7 +1,13 @@
-// TODO: integrate with Sentry?
+import * as Sentry from '@sentry/react';
+import { sentry } from '../../../config/secrets';
+
 export function reportClientSideError(
   e: Error,
-  message = 'Unhandled error'
-): void {
-  return console.error(`Client: ${message}`, e);
+  message = sentry.dns === 'dsn_from_sentry_dashboard'
```

Comment:
Can we fix this in `/config/secrets.js`? That is, make it `sentry.dsn`?

Reply:
My typo has survived two years so far, but sure.

Reply:
I propose `sentry.dsn$`

Reply:
```js
$(function() {
 Sentry.init({
    dsn: sentry.dsn$,
    integrations: [new Integrations.BrowserTracing()],
    tracesSampleRate: 1.0
  });
});
```